app/src/test/java/com/github/swent/echo/viewmodels/HomeScreenViewModelTest.kt

```diff
@@ -0,0 +1,118 @@
+package com.github.swent.echo.viewmodels
+
+import com.github.swent.echo.compose.map.MAP_CENTER
+import com.github.swent.echo.data.model.Association
+import com.github.swent.echo.data.model.Event
+import com.github.swent.echo.data.model.EventCreator
+import com.github.swent.echo.data.model.Location
+import com.github.swent.echo.data.model.Tag
+import com.github.swent.echo.data.repository.Repository
+import com.github.swent.echo.fakes.FakeAuthenticationService
+import io.mockk.mockk
+import java.time.ZonedDateTime
+import junit.framework.TestCase
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.StandardTestDispatcher
+import kotlinx.coroutines.test.TestCoroutineScheduler
+import kotlinx.coroutines.test.setMain
+import org.junit.Before
+import org.junit.Test
+
+class HomeScreenViewModelTest {
+
+    private val fakeAuthenticationService = FakeAuthenticationService()
+    private val mockedRepository = mockk<Repository>(relaxed = true)
+    private lateinit var homeScreenViewModel: HomeScreenViewModel
+    private val scheduler = TestCoroutineScheduler()
+
+    @OptIn(ExperimentalCoroutinesApi::class)
+    @Before
+    fun init() {
+        fakeAuthenticationService.userID = "u0"
+        Dispatchers.setMain(StandardTestDispatcher(scheduler))
+        runBlocking {
+            homeScreenViewModel = HomeScreenViewModel(mockedRepository, fakeAuthenticationService)
+        }
+        scheduler.runCurrent()
+    }
+
+    @Test
+    fun overlayTest() {
+        TestCase.assertEquals(homeScreenViewModel.overlay.value, Overlay.NONE)
```

Comment:
I think the standard `assertEquals` should be imported from `import org.junit.Assert.*`... Probably this `assertEquals` from `TestCase` does the same but I think to also be more consistent with the other test we should use the first one.

Reply:
True I'll change that